config/routes.rb

```diff
@@ -43,11 +43,7 @@
     get "/contact-details", to: "contact_details#show"
     post "/contact-details", to: "contact_details#submit"
 
-    # Question 8.0: Do you know your NHS number?
-    get "/know-nhs-number", to: "know_nhs_number#show"
-    post "/know-nhs-number", to: "know_nhs_number#submit"
```

Comment:
Annoyingly, removing the routes to the page will break the form for people already on the `/know-nhs-numbers` page and the previous page.

E.g. I am a user on the `/know-nhs-number` page during a deploy. I submit the form during or after the deploy, which will result in a 404 error.

I wonder if we could solve that for this one, by adding a redirect to the NHS number page for both of these routes, then remove them later, in an A AB B style deploy?

Reply:
Thanks @bilbof - nice catch thanks!  I've tested these manually, and as they are temporary - I guess we won't need an automated test for these.  Shout if we do though.